docs/reference/builder.md

```diff
@@ -937,6 +943,13 @@ All new files and directories are created with a UID and GID of 0.
 > If you build using STDIN (`docker build - < somefile`), there is no
 > build context, so `COPY` can't be used.
 
+Optionally `COPY` accepts a flag `--from=[name|index]` that can be used to set
+the source location to a previous build stage(created with `FROM .. AS <name>`
```

Comment:
- add a space before `(`
- "created with _the_ ..." (or remove "command" on the next line)